### PR TITLE
Fix/change password padding

### PR DESCRIPTION
**Pull request by ektaghag-eaton**
Fixes # .

#### Changes proposed in this Pull Request:

- Change Password Padding Fix
- Fix Slots and SlotProps
-
-

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

- Update Change password dialog in example with following code
<ChangePasswordDialog
            open={app.showChangePasswordDialog}
            onPrevious={(): void => app.setShowChangePasswordDialog(false)}
            onSubmit={(): void => app.setShowChangePasswordDialog(false)}
            onFinish={(): void => logOut()}
            slots={{
                SuccessScreen: (props): JSX.Element => (
                    <Box>
                        {props?.icon}
                        {props?.messageTitle}
                    </Box>
                ),
            }}
            slotProps={{
                SuccessScreen: {
                    icon: <Alarm />,
                    messageTitle: 'Yeah',
                    message: 'ee',
                },
            }}
        />

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Comment by ektaghag-eaton** [comment]
@surajeaton , and @Manoj1907 do not merge after approval. Need to remove test code

File: login-workflow/example/src/components/ChangePassword.tsx
Comment (surajeaton) [inline login-workflow/example/src/components/ChangePassword.tsx:1]: There are unused imports in this, which will fail lint.Please remove

Comment (ektaghag-eaton) [reply login-workflow/example/src/components/ChangePassword.tsx:1]: Done please review .

**Review by surajeaton — APPROVED** [review]
Looks good

**Pull request merged**
